src/main/java/ua/danit/final_project/services/tasks/RepeatableTaskExecutor.java

```diff
@@ -0,0 +1,107 @@
+package ua.danit.final_project.services.tasks;
+
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import ua.danit.final_project.entities.Task;
+import ua.danit.final_project.repositories.TaskRepository;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@Service
+@EnableScheduling
+public class RepeatableTaskExecutor {
+
+  private static final long UPDATE_PERIOD = 1000 * 60 * 5;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Cast one of the operands of this multiplication operation to a "long". ([squid:S2184](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS2184))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=109&issues=AWXI83-p_eEZPtUmp6G1&open=AWXI83-p_eEZPtUmp6G1)

[ISSUE_KEY]: AWXI83-p_eEZPtUmp6G1